src/protocol.h

```diff
@@ -354,34 +354,45 @@ static inline bool MayHaveUsefulAddressDB(ServiceFlags services)
     return (services & NODE_NETWORK) || (services & NODE_NETWORK_LIMITED);
 }
 
+enum class AddrSerialization {
```

Comment:
nit, feel free to ignore: since this is also used in **un**serialization, the name could be misleading. `AddrFormat`?

Reply:
I've changed it to `CAddress::Format`.